### PR TITLE
Evgen.starostenko/homework2

### PR DESCRIPTION
**Pull request by evgenstarostenkoo**

### CODE REVIEW

File: cypress/e2e/Tests/sign_in_test.spec.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+const {sign_in_page} = require("../selectors/sign_in_page");
+const myRandomString = () => {
+    let text = "";
+    let possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+
+    for (let i = 0; i < 10; i++)
```
Comment (ievgendco) [inline cypress/e2e/Tests/sign_in_test.spec.js:6]: Please migrate it to the separate method in helper

File: cypress/e2e/Tests/sign_in_test.spec.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    })
+
+    it('should show "Real World App logo"', () => {
+        cy.get(sign_in_page.logo_image).should('be.visible').and('have.attr', 'xmlns', 'http://www.w3.org/2000/svg')
+    });
+
+    it('should show "Sign in" title', () => {
+        cy.get(sign_in_page.title_text).should('be.visible').and('have.text', 'Sign in')
+    });
+    it('Show typeable "UserName" field', () => {
```
Comment (ievgendco) [inline cypress/e2e/Tests/sign_in_test.spec.js:25]: Add one space between tests

**Review by ievgendco — CHANGES_REQUESTED** [review]

File: cypress/support/commands.js [hunk 3]
```diff
@@ -22,4 +22,15 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+// Random string
+
+export const myRandomString = () => {
```
Comment (anatolygusakovsky1) [inline cypress/support/commands.js:29]: Do not store non-cypress functions in cypress/support/commands.js.
Feel free to create new file/folder+file in cypress directory and put it there

**Review by anatolygusakovsky1 — CHANGES_REQUESTED** [review]

File: cypress/support/commands.js (hunk 3 above)
Comment (evgenstarostenkoo) [reply cypress/support/commands.js]: Fixed, please review

**Review by anatoly-devico — APPROVED** [review]
Approved

**Pull request merged**